tools/version.rb

```diff
@@ -51,15 +51,11 @@
 versions = []
```

Comment:
Bump version here too?

Reply:
I assume you meant at the top of the tools/version.rb file, which I have now done.